openBCIBoard.js

```diff
@@ -708,14 +845,15 @@ function OpenBCIFactory() {
 
             if (!pInput && !nInput) {
                 this.impedanceTest.active = false;
-                this.writeOutDelay = k.OBCIWriteIntervalDelayMSShort;
+                //this.writeOutDelay = k.OBCIWriteIntervalDelayMSShort;
             } else {
-                this.writeOutDelay = k.OBCIWriteIntervalDelayMSLong;
+                //this.writeOutDelay = k.OBCIWriteIntervalDelayMSLong;
```

Comment:
Ah so this is actually critical because this tells the channel to essentially not inject the test signal on either channel, in other words, it stops testing on either P or N. Does this make sense?